dlib/CMakeLists.txt

```diff
@@ -353,7 +356,10 @@ if (NOT TARGET dlib)
       endif()
 
       if (DLIB_JPEG_SUPPORT)
-         # try to find libjpeg 
+         # try to find libjpeg
+         if(HUNTER_ENABLED) 
+            hunter_add_package(JPEG)
```

Comment:
It's `hunter_add_package(Jpeg)` (case matters), see [pkg.jpeg](https://github.com/ruslo/hunter/wiki/pkg.jpeg).